### DISCUSSION TITLE
What is the unit of potential energy (epot)?

### DISCUSSION THREAD

**Discussion by hu-mingyu** [ramses-organisation/ramses#55] [Q&A] [answered]
Hi, does anyone know the unit or a clear definition of potential energy in Ramses? I couldn’t find it in the documentation. 

Here’s what I’ve checked so far:  
- `poisson/force_fine.f90` includes the relevant parameters `epot_loc` and `epot_all`.  
- `poisson/poisson_parameters.f90` doesn't mention `epot`.

Thanks in advance!

**Comment by nbrucy** [comment, marked as answer]
Hi,

The units are [code units](https://ramses-organisation.readthedocs.io/en/dev/wiki/Physics.html#units-parameters) for energy (`unit_density` $\times$ `unit_length`$^5$ / `unit_time`$^2$).

**Reply by nbrucy** [reply]
Also check #56 if you are parsing the logfile for the energies.
